ui/app/src/components/DashletCard/DashletCard.tsx

```diff
@@ -23,6 +23,14 @@ import CardContent, { CardContentProps } from '@mui/material/CardContent';
 import Box, { BoxProps } from '@mui/material/Box';
 import { UNDEFINED } from '../../utils/constants';
 import CardActions from '@mui/material/CardActions';
+import OpenInFullOutlinedIcon from '@mui/icons-material/OpenInFullOutlined';
+import CloseFullscreenOutlinedIcon from '@mui/icons-material/CloseFullscreenOutlined';
+import IconButton from '@mui/material/IconButton';
+import useSiteDashboardContext from '../SiteDashboard/useSiteDashboardContext';
+import Accordion from '@mui/material/Accordion';
+import AccordionDetails from '@mui/material/AccordionDetails';
+import ExpandMoreIcon from '@mui/icons-material/KeyboardArrowDown';
+import AccordionSummary from '@mui/material/AccordionSummary';
```

Comment:
Rollback the use of accordion. Just wrap the body area in a conditonal. Can try a [transition component](https://mui.com/material-ui/transitions/) (e.g. Grow) or keep it simple. Also, don't add an icon for the collapsing for _desktop_.

Alternatively, rather use the Accordion alone — no card components. Accordion is a bit bloated in styles and features so think easiest to stick with card for now.